### ISSUE TITLE
Downshift for tables (inputs without label)

### ISSUE THREAD

**Issue by janhesters** [kentcdodds/ama#638]
Hi Kent,

Another day, another question 😊 

**TL;DR:** Is there an aria compliant way to use Downshift for an input without a label (or hide the label)? I'm using inputs as rows for tables. They don't have labels because labels would clutter up the table (see screenshot).

![Screenshot 2019-03-21 at 14 19 38](https://user-images.githubusercontent.com/31096420/54754687-60ca2d00-4be4-11e9-9e53-47708e1117f4.png)

**Long version:**

In the table I render an input for all non-constant values of contacts that get passed to it. These inputs can be edited on the fly. If the user did that, he hits the save button (see screenshot on the left).

```js
function ContactRow({ contact: original, handleContactSave }) {
  const [contact, setContact] = useState(original);
  const isNewContact = original.id === 'new-contact';

  const renderOption = value => (
    <option key={`${contact.id}-${value}`} value={value}>
      {strings[value]}
    </option>
  );

  const renderBodyCell = key => {
    const value = contact[key];
    const testId = `contact-${key}${isNewContact ? '-new-contact' : ''}`;
    const handleChange = e => {
      e.preventDefault();
      setContact({ ...contact, [key]: e.target.value });
    };
    return (
      <td key={`${key}-${contact.id}`}>
        // groups and tendencies are arrays with constant values
        {includes(value, [...groups, ...tendencies]) ? (
          <select value={value} data-testid={testId} onChange={handleChange}>
            {includes(value, groups)
              ? map(renderOption, groups)
              : map(renderOption, tendencies)}
          </select>
        ) : (
          // all other values are strings and should by editable on the fly
          <input value={value} data-testid={testId} onChange={handleChange} />
        )}
      </td>
    );
  };

// ... more stuff
```

I'm in the process of adding autocomplete. But, since there are no labels, I'm not sure if the table will be aria compliant. Or should I render labels but give them the style `display: 'none'`?

If you could help me out, thank you very much 😊

**Comment by kentcdodds** [comment]
Use a label for the headings and [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) to point to that one.

There may be more needed to identify the row, but I'm not sure.
